synapse/rest/admin/rooms.py

```diff
@@ -745,13 +745,19 @@ async def on_GET(
 
         time_now = self.clock.time_msec()
         results["events_before"] = await self._event_serializer.serialize_events(
-            results["events_before"], time_now
+            results["events_before"],
+            time_now,
+            bundle_aggregations=True,
```

Comment:
As these are admin APIs we can do whatever we want, but the intention is for them to match the Client-Server APIs, so I gave them the same behavior!